application/src/main/kotlin/com/henriquebarucco/ssbc/sensor/get/dto/GetSensorOutput.kt

```diff
@@ -0,0 +1,25 @@
+package com.henriquebarucco.ssbc.sensor.get.dto
+
+import com.henriquebarucco.ssbc.sensor.Sensor
+import com.henriquebarucco.ssbc.sensor.dto.SensorPageDto
+
+
+data class GetSensorOutput(
+    val content: List<Sensor>?,
```

Comment:
Esse caso aqui não precisa ser opicional, a uníca diferença que esperamos é que seja uma lista vazia.

```suggestion
    val content: List<Sensor>,
```